proto/cosmos/staking/v1beta1/tx.proto

```diff
@@ -4,6 +4,7 @@ package cosmos.staking.v1beta1;
 import "gogoproto/gogo.proto";
 import "cosmos/base/v1beta1/coin.proto";
 import "cosmos/staking/v1beta1/staking.proto";
```

Comment:
since rc0 is released, should we bump to v1beta2?

Reply:
I don't know ;)

Reply:
I mean - this should be a separate PR, right?

Reply:
See: 
```shell
$ make proto-check-breaking
proto/cosmos/staking/v1beta1/tx.proto:1:1:Field "6" on message "MsgCreateValidator" changed type from "string" to "message".
```
This is considered a breaking change so I believe this should be part of this PR.

Reply:
I think we are trying to get this into 0.40 (it's labeled as such) so the breaking change is fine.

Reply:
I guess not then if we get this into 0.40
see more about proto naming and convention in ADR 023 on why we keep `beta`: https://github.com/cosmos/cosmos-sdk/blob/master/docs/architecture/adr-023-protobuf-naming.md#guidelines-on-unstable-alpha-and-beta-package-versions